src/components/events/partials/ModalTabsAndPages/EditScheduledEventsSummaryPage.tsx

```diff
@@ -195,6 +196,7 @@ const EditScheduledEventsSummaryPage = <T extends RequiredFormProps>({
 					<div className="row">
 						{/* Show only if there no changes*/}
 						<div className="alert sticky warning">
+							<LuTriangleAlert className="warning-symbol-warning"/>
```

Comment:
```suggestion
                        <div className="alert sticky warning" style={{ display: "flex", alignItems: "center" }}>
							<LuTriangleAlert className="warning-symbol-warning"/>
```

or just factor out this and the previous instance.